array.c

```diff
@@ -4292,22 +4302,42 @@ ary_resize_smaller(VALUE ary, long len)
 
 /*
  *  call-seq:
- *     ary.delete(obj)            -> item or nil
- *     ary.delete(obj) {block}    -> item or result of block
+ *    array.delete(obj) -> deleted_object
+ *    array.delete {|obj| ... } -> deleted_object or block_return
+ *
+ *  Removes zero or more elements from +self+; returns +self+.
+ *
+ *  ---
+ *
+ *  When no block is given,
+ *  removes from +self+ each element +ele+ such <tt>that ele == obj</tt>;
```

Comment:
I think `that` jumped inside the `<tt>` here

Reply:
_That_s are like that!

Reply:
Fixed in next PR.